### PR TITLE
Install dependencies in one click on Windows

### PR DESCRIPTION
**Pull request by hassanila**
This is a batch file for Windows that check if nodejs is installed, if not it redirects user to the download else it will install the dependencies right away...

### CODE REVIEW

File: Install Dependencies (Windows).bat [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+::NET SESSION >nul 2>&1
+::IF %ERRORLEVEL% NEQ 0 (
+::	echo This setup needs Admin permissions. Please run this file as an Admin.
+::      echo Press any key to exit...
+::	pause >nul 2<&1
+::	exit
+::)
+
+where nodem >nul 2>&1
```
Comment (hassanila) [inline Install Dependencies (Windows).bat]: Sorry but line 15 is wrong it needs to be node instead of nodem. It's a typo :+1:

**Comment by davidmann4** [comment]
good idea! anyone tested this?

**Comment by davidmann4** [comment]
plz someone test this

**Comment by davidmann4** [comment]
did not worked?

**Comment by hassanila97** [comment]
@DavidMann4 it works but I changed it little bit will reopen a new pull req... I dont know how this patches work  hhh   Im new to GtiHub... will do it tomorrow

**Pull request closed without merging**